src/app/editor.js

```diff
@@ -2,10 +2,32 @@
 
 var EventManager = require('../lib/eventManager')
 
+var csjs = require('csjs-inject')
 var ace = require('brace')
 var Range = ace.acequire('ace/range').Range
 require('../mode-solidity.js')
 
+var css = csjs`
```

Comment:
Oh by the way - the name is flawed, because one esential feature of `csjs` is to end "global namespace pollution" and thus preventing styles from leaking and crashing styles of other components written by other people.

It also changes the "seperation of technology" (html,css,js) into "separation of concerns" by co-locating (html,css,js) of one component (which is anyway heavily co-dependent by nature) in the same file, which is awesome for authors who want to search for everything (styles, markup and behavior) related to a single e.g. ButtonComponent.

Without that, the author would need to search different files and scroll through a lot of unrelated code or would need to make heavy use of editor assisted search functionality.